iot/iot-device-registry-infinispan/src/main/java/io/enmasse/iot/registry/infinispan/server/DeviceRegistryRestServer.java

```diff
@@ -3,10 +3,12 @@
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
 
-package io.enmasse.iot.registry.infinispan;
+package io.enmasse.iot.registry.infinispan.server;
 
 import org.eclipse.hono.config.ServiceConfigProperties;
 import org.eclipse.hono.service.http.HttpServiceBase;
+import org.springframework.stereotype.Component;
 
+@Component
 public class DeviceRegistryRestServer extends HttpServiceBase<ServiceConfigProperties> {
 }
```

Comment:
new line

Reply:
I am not sure what to remove here.

Reply:
It's just showing that there's no new line at the end of file ... nitpicking :)

Reply:
Oh ok … fixed!